appengine/src/main/java/com/google/googleinterns/gscribe/resources/io/exception/InvalidAuthorizationRequestException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.googleinterns.gscribe.resources.io.exception;
+
+import javax.ws.rs.BadRequestException;
+
+public class InvalidAuthorizationRequestException extends BadRequestException {
```

Comment:
Create a single InvalidRequestException for InvalidAuthorizationRequestException, InvalidIDTokenException, MissingRequestParametersException, provide appropriate message while throwing the exception.